src/components/styles/input.css

```diff
@@ -0,0 +1,69 @@
+.form-container {
+  display: flex;
+  flex-direction: column;
+  gap: 1.88rem;
+  width: 100%;
+  max-width: 800px;
+  margin: 3rem auto 10em;
+  padding: 2.6rem 2rem;
+  border-top: 1px solid var(--gray-three);
+}
```

Comment:
- [ ] Hello! 🚀 Great job on implementing the styling for the add form. 🎨 However, I've noticed a minor issue. In the design, the form elements are aligned to the left, but in your implementation, they appear to be centered. 🧐 Let's make sure we stick to the design as closely as possible. Here's a comparison:

👁️‍🗨️ Your Design:
![Screenshot from 2023-09-28 15-25-22](https://github.com/Izebiafe/book-store/assets/112261237/8888d3d6-0791-4b35-8198-1d35a94cf174)

👁️‍🗨️ Zeplin Design:
![Screenshot from 2023-09-28 15-24-54](https://github.com/Izebiafe/book-store/assets/112261237/d0878873-45ff-495a-b085-df925cf6425a)

- [ ] To align with the Zeplin design, we should ensure that the form elements are left-aligned.

Additionally, please implement a right border for the "Comments" section and remove the button.

Here's a breakdown of the changes:

Alignment: Make sure that the form elements are left-aligned to match the Zeplin design.

Border for Comments: Add a right border to the `comments and remove` button to match the design.

Feel free to reach out if you have any questions or need further assistance! 🤝